DNN Platform/Library/Services/Upgrade/DnnInstallLogger.cs

```diff
@@ -67,11 +63,11 @@ public static void InstallLogInfo(object message)
 
 		private static void LogInstaller(string logType, string message)
 		{
-			string logFile = "InstallerLog" + DateTime.Now.Year.ToString(CultureInfo.InvariantCulture) + DateTime.Now.Month.ToString(CultureInfo.InvariantCulture) + DateTime.Now.Day.ToString(CultureInfo.InvariantCulture) + ".resources";
-			string logfilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Portals\_default\Logs\" + logFile);
+		    var logFile = InstallController.Instance.InstallerLogName;
+			var logfilePath = Path.Combine(AppDomain.CurrentDomain.BaseDirectory, @"Portals\_default\Logs\" + logFile);
 			using (var writer = new StreamWriter(logfilePath, true))
 			{
-				writer.WriteLine(DateTime.UtcNow.ToString(CultureInfo.InvariantCulture) + " " + logType + " " + CallingType + " " + message);
+				writer.WriteLine(DateTime.Now.ToString(CultureInfo.InvariantCulture) + " " + logType + " " + CallingType + " " + message);
```

Comment:
I did this to be the same as the regular log files. I heard it from QA that they had problem relating bugs to actual lines between the two log files (regular and installation). IMO, both should have the same format; either both are UTC or local time but not one UTC and the other local.